libs/output-mapping/tests/Needs/NeedsTestTables.php

```diff
@@ -8,7 +8,9 @@
 
 #[Attribute] class NeedsTestTables
 {
-    public function __construct(public readonly int $count = 1)
-    {
+    public function __construct(
+        public readonly int $count = 1,
+        public readonly bool $typedTable = false,
+    ) {
     }
 }
```

Comment:
Původně jsem myslel, že to budu potřebovat pro testy. Až pak jsem si všimnul, že v `TableStructureValidatorTest` se pracuje s mockama sapi clienta. Zmátlo mě že to dědí z `AbstractTestCase`. Zatím to tu nechávám, určitě se to bude někde hodit.